sdk/src/dataMapper/AttestationDataMapper.ts

```diff
@@ -48,14 +48,48 @@ export default class AttestationDataMapper extends BaseDataMapper<
     orderDirection?: OrderDirection,
   ) {
     const attestations = await super.findBy(first, skip, where, orderBy, orderDirection);
-    attestations.forEach(async (attestation) => {
-      const schema = (await this.veraxSdk.schema.getSchema(attestation.schemaId)) as Schema;
-      attestation.decodedPayload = decodeWithRetry(schema.schema, attestation.attestationData as `0x${string}`);
-    });
+    await Promise.all(
+      attestations.map(async (attestation) => {
+        await this.enrichAttestation(attestation);
+      }),
+    );
 
     return attestations;
   }
 
+  private async enrichAttestation(attestation: Attestation) {
+    const schema = (await this.veraxSdk.schema.getSchema(attestation.schemaId)) as Schema;
+    attestation.decodedPayload = decodeWithRetry(schema.schema, attestation.attestationData as `0x${string}`);
+    // Check if data is stored offchain
+    if (attestation.schemaId === "0xa288e257097a4bed4166c002cb6911713edacc88e30b6cb2b0104df9c365327d") {
```

Comment:
Make `0xa288e257097a4bed4166c002cb6911713edacc88e30b6cb2b0104df9c365327d` a constant in `utils/constants.ts`